### DISCUSSION TITLE
256 цвет терминала в консоли (xterm console 256 color scheme)

### DISCUSSION THREAD

**Discussion by johnrembo** [elfmz/far2l#1404] [Q&A]
Проблема такая.  Vim (ну или NeoVim) из консоли фара запускается нормально, но что-то происходит странное с цветами. Vim настроен на 256 цветов (:set t_Co=256) и под соответствующую тему. Как будто внутри консоли обрезаются цвета до 8. 

![Screenshot_20221112_113446](https://user-images.githubusercontent.com/26782068/201452687-5d7dccfb-dd7b-4e08-aec4-ee01abd08625.png)

А вот tput color показывает 8, и xterm вроде как должен быть xterm-256color? Причем например с самим bash ничего подобного не наблюдается - нормально листает в цвете и в таком каком надо, да и сам менеджер понимает 256 цветную палитру без проблем.

![Screenshot_20221112_113051](https://user-images.githubusercontent.com/26782068/201452522-250ad800-9bbb-43db-8b0b-afc859979adc.png)

Если переключить vim в базовый режим, как-нибудь так, то цвета появляются и в нем, но это, мне кажется, такое себе решение.

```
vim -c "set t_Co=16" -c "colorscheme slate" -c "AirlineTheme base16" -c "set nocursorline" !.!
```
![Screenshot_20221112_114010](https://user-images.githubusercontent.com/26782068/201452905-b4622268-0cef-4065-b70b-b931c767f41a.png)

Хотелось бы конечно полноцветный вариант.

Может я чего-то не нашел в документации или трекере? Может какой-то есть простой способ, который просто настолько очевиден, что его не надо было включать в инструкции? 

Чисто мнение: Сейчас у меня слом парадигмы. Вроде как фар сила, но всё-таки для программирования встроенного редактора не хватает. Поэтому, вроде как нужен, вим, а его подружить с фаром как-то не очень получается. Из-за этого всё больше склоняюсь к варианту vim-fm (и сделать его похожим на фар), но он как менеджер конечно ерунда. А если бы фар подружить с zsh, neovim, и прочим хозяйством, вот это было бы бесценно.

**Comment by elfmz** [comment]
так у вас старая версия far2l которая не умеет в 256 цветов, обновитесь до текущего мастера

**Reply by johnrembo** [reply]
О, всё заработало! Спасибо за Ваш труд!

**Reply by johnrembo** [reply]
А, нет, что-то поторопился. Одинаково оказывается, так же пишет, что доступно только 8 цветов.

```
--------------------------------
/opt/far2l/_build$ tput colors
8
--------------------------------
```

Причем если запустить `far2l --tty` в например 'konsole' то `zsh` вроде как нормально отображается, а сам по себе никак, всё в 8 цветах. Может быть я его как-то не так запускаю?

**Reply by johnrembo** [reply]
Сорян, опять поторопился.
Проблема была из-за того что я строго по инструкции 2.4.1 чекнул, а надо было с мастера ставиться

**Reply by johnrembo** [reply]
![image](https://user-images.githubusercontent.com/26782068/201807804-d9205e4a-d0e8-4f55-8b4a-2b4883e62016.png)

Огонь
